docs/scos/dev/feature-integration-guides/202212.0/inventory-management-feature-integration.md

```diff
@@ -0,0 +1,705 @@
+---
+title: Inventory Management feature integration
```

Comment:
- must be done in pbc folder as https://docs.spryker.com/docs/pbc/all/warehouse-management-system/202212.0/install-and-upgrade/install-features/install-the-inventory-management-feature.html but in 202304.0